core/include/seqan/align/evaluate_alignment.h

```diff
@@ -252,6 +252,241 @@ TScoreVal computeAlignmentStats(Align<TGaps, TAlignSpec> const & align,
     return computeAlignmentStats(stats, align, scoringScheme);
 }
 
+// ----------------------------------------------------------------------------
+// Function alignmentIdentityString()
+// ----------------------------------------------------------------------------
+
+/*!
+ * @fn Align#alignmentIdentityString()
+ * @headerfile <seqan/align.h>
+ * @brief Returns a CharString of the similarity of positions in an alignment
+ *
+ * @signature CharString alignmentIdentityString();
```

Comment:
Please add the complete signature using the described params below.
Optional arguments are put in brackets like: signature(mandatory [, optionalA] [, optionalB]);